src/client/components/app/Transactions/WithdrawTx.tsx

```diff
@@ -193,7 +196,14 @@ export const WithdrawTx: FC<WithdrawTxProps> = ({ header, onClose, children, ...
   };
 
   const approve = async () => {
-    await dispatch(VaultsActions.approveZapOut({ vaultAddress: selectedVault.address }));
+    try {
```

Comment:
what could possibly fail here that we need a try/catch?

Reply:
User rejecting to sign, wallet error, timeout ...